modules/01_hello_world_api/README.md

```diff
@@ -0,0 +1,62 @@
+# Hello world python web API
+
+In the first stage we're going to create a python web API with [Flask](https://flask.palletsprojects.com/en/2.2.x/quickstart/) that will return "Hello world" on the "/" page.
+
+1. Add and install `Flask` python package.
+
+    >To be able to build the API, we first need to get the package of API we'll be using.
+
+    1. Create a `requirements.txt` file in the root of the project folder
+
+        ```sh
+        touch requirements.txt
+        ```
+
+    2. Modify the contents of the `requirements.txt` with:
```

Comment:
```suggestion
    2. Add the following content to `requirements.txt`:
```

I was confused for a few seconds, thinking how calling "flask" would modify the file :)